metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -262,7 +263,16 @@ def process_dataflow_node(
 
             else:
 
-                raise ValueError(f"Unrecognized Glue data object type: {node_args}")
+                if self.source_config.ignore_unsupported_connectors:
+                    
+                    self.report.report_warning(
```

Comment:
does `report_warning` create a possibility of the pipeline failing if pipeline warnings == failure setting is true? 
Is that the default?

Reply:
Good point – I'll use logging.log instead